### ISSUE TITLE
CIFuzz: checks currently failing (cannot check out current state)

### ISSUE THREAD

**Issue by jwhui** [google/oss-fuzz#3842]
```
Step 5/6 : WORKDIR openthread
 ---> Running in 53c7da1fc298
Removing intermediate container 53c7da1fc298
 ---> 8b73e2a90f6c
Step 6/6 : COPY build.sh *.options $SRC/
 ---> 69e0cb6092e8
Successfully built 69e0cb6092e8
Successfully tagged gcr.io/oss-fuzz/openthread:latest
2020-05-18 15:06:08,380 - root - DEBUG - Stderr of command 'git clone https://github.com/openthread/openthread openthread' is Cloning into 'openthread'...
.
2020-05-18 15:06:08,584 - root - DEBUG - Stderr of command 'git fetch origin refs/heads/disable-copy' is fatal: Couldn't find remote ref refs/heads/disable-copy
.
2020-05-18 15:06:08,584 - root - ERROR - Can not check out requested state.
2020-05-18 15:06:08,584 - root - ERROR - Error building fuzzers for project openthread with pull request refs/heads/disable-copy.
Running: docker build -t gcr.io/oss-fuzz/openthread projects/openthread
```

Full log: https://github.com/openthread/openthread/runs/685753508?check_suite_focus=true
PR: https://github.com/openthread/openthread/pull/4973

Thoughts?

**Comment by inferno-chromium** [comment]
Really sorry, we are tracking this in #3838. You should be unblocked by https://github.com/google/oss-fuzz/pull/3844, can you verify if your CIFuzz works.

**Comment by jwhui** [comment]
@inferno-chromium , thanks for the response. I tried re-running the GitHub Actions job, but the same failure noted in the original post above occurs. Thoughts?

**Comment by inferno-chromium** [comment]
Ah, i confused this with another fire i was trying to fix.

"Couldn't find remote ref refs/heads/disable-copy" feels like some git issue, you think this is CIFuzz related ?

**Comment by evverx** [comment]
It seems to be a duplicate of https://github.com/google/oss-fuzz/issues/3732. Currently the "rerun jobs" button can't be used to restart failed jobs. PRs have to be force-pushed instead.

**Comment by jwhui** [comment]
Yes, we were initially experiencing #3838 and now experiencing #3732. Force-pushing resolves our current issues.
